chimedb/data_index/orm.py

```diff
@@ -619,12 +608,19 @@ class ArchiveFileCopy(base_model):
         - 'M': maybe, can delete if we need space
         - 'N': no, attempt to delete
         In all cases we try to keep at least two copies of the file around.
+    size_b : integer
+        Allocated size of file in bytes (calculated as a multiple of 512-byte
```

Comment:
The 512-byte thing is Linux-specific.  Is it relevant?  Do we need to say anything about how the size is calculated?

Reply:
`size_b` is calculated as:
```
        # If the file exists calculate its md5sum and check against the DB
        if os.path.exists(fullpath):
            if util.md5sum_file(fullpath) == fcopy.file.md5sum:
                log.info("File is A-OK!")
                fcopy.has_file = "Y"
                copy_size_b = os.stat(fullpath).st_blocks * 512
                fcopy.size_b = copy_size_b
            else:
                log.error("File is corrupted!")
                fcopy.has_file = "X"
        else:
            log.error("File does not exist!")
            fcopy.has_file = "N"

```
where `st_blocks` is the no. of 512-byte blocks allocated for the file. I think the part in brackets is saying how it is calculated.

Reply:
Right, I know that.  But we only do that (multiply by 512) because that's how POSIX says `stat(3)` should work.  Why bother pointing it out to the user?  Do they care?  What are they going to do with that information?

Reply:
Oh I see what you mean, I'll remove it